elements/pfe-navigation-account/src/pfe-navigation-account.js

```diff
@@ -98,6 +280,10 @@ class PfeNavigationAccount extends PFElement {
 
   connectedCallback() {
     super.connectedCallback();
+
+    if (this.hasAttribute("lang")) {
```

Comment:
The properties system will manage this for you so it won't be necessary any more.  Example:

```
static get properties() {
    return {
      _lang: {    // underscore is required because `lang` is a global HTML attribute
        title: "Language support",
        type: String,
        default: "en",    // <-- Can I recommend `en-US` to be more accurate?
        observer: "_langHandler" // <-- method for managing the update of the lang property
      },
```

Reply:
Would like to keep this prerelease friendly.